index.js

```diff
@@ -21,15 +21,10 @@ fontmachine.makeGlyphs = function(opts, callback) {
     var stack = [];
     fontnik.load(opts.font, function(err, faces) {
         if (err) return callback(err);
-        // temp workaround: fill this out
+
```

Comment:
Thanks @springmeyer! Opened https://github.com/mapbox/fontmachine/issues/2 to track this further.